zanata-war/src/main/resources/messages.properties

```diff
@@ -1178,18 +1177,18 @@ jsf.RequestJoinGroup=Request to add project version to group
 jsf.AlreadyInGroup=Already in Group
 jsf.email.joingrouprequest.AdditionalInfoMessage=To ensure your request is processed without delay, please provide any additional information that will help the group maintainers to process your request.
 !FIXME make key similar to related keys
-jsf.RequestToJoinLanguageTeamTitle=Request To Join "{0}" Language Team
-jsf.RequestRoleLanguageTeamTitle=Request Role in "{0}" Language Team
-
+jsf.email.RequestToJoinLanguageTeam.title=Request to join {0} [{1}] language team
```

Comment:
This needs a source comment that says what `{0}` and `{1}` will be replaced with. It is ambiguous enough that translators should not be left to guess this.